daemons/gptp/common/ieee1588.hpp

```diff
@@ -441,8 +449,10 @@ static inline void TIMESTAMP_ADD_NS( Timestamp &ts, uint64_t ns ) {
  * Provides a generic interface for hardware timestamping
  */
 class HWTimestamper {
+
 protected:
 	uint8_t version; //!< HWTimestamper version
+	struct phy_delay delay;
```

Comment:
Please add something like:
//!< strruct phy_delay member containing the phy delay information